compiler/rustc_target/src/target_features.rs

```diff
@@ -49,6 +52,7 @@ impl<CTX> HashStable<CTX> for Stability {
             Stability::Forbidden { reason } => {
                 reason.hash_stable(hcx, hasher);
             }
+            Stability::TargetModifierOnly { .. } => {}
```

Comment:
Not hashing the fields here is sus... but I'm anyway going to entirely remove this variant in my PR.